lib/commands/execute.js

```diff
@@ -0,0 +1,38 @@
+import _ from 'lodash';
+import { errors, BaseDriver } from 'appium-base-driver';
+import logger from '../logger';
+
+let extensions = {};
+
+extensions.execute = async function execute (script, args) {
```

Comment:
Execute is a pretty vague file name. What is this doing exactly?

Reply:
I am following the structure they setup in other drivers:
https://github.com/appium/appium-android-driver/blob/master/lib/commands/execute.js

Reply:
The general command is called execute script: http://appium.io/docs/en/commands/mobile-command/